media_management_api/media_service/models.py

```diff
@@ -1,7 +1,6 @@
 from django.db import models, transaction, Error
 from django.db.models import Max
 from django.contrib.auth.models import User
-from django.contrib.postgres.fields import JSONField
```

Comment:
Nice. This was [deprecated in 3.1](https://docs.djangoproject.com/en/3.2/releases/3.1/#postgresql-jsonfield).